src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -192,13 +194,13 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
         isResizable: true,
         name: messages['Admin consent required'],
         fieldName: 'isAdmin',
-        minWidth: (tokenPresent) ? 150 : 200,
-        maxWidth: (tokenPresent) ? 200 : 300,
+        minWidth: (tokenPresent && permissionModeType === DISPLAY_DELEGATED_PERMISSIONS) ? 150 : 200,
+        maxWidth: (tokenPresent && permissionModeType === DISPLAY_DELEGATED_PERMISSIONS) ? 200 : 300,
```

Comment:
maybe be like 
`const currentPermType = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;` and replace all instances of `permissionModeType === DISPLAY_DELEGATED_PERMISSIONS` with `currentPermType`?

it might be `let currentPermType = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;` but try  `const` first.